src/pom.xml

```diff
@@ -2506,7 +2506,7 @@
               <fork>false</fork>
               <compilerArgs>
                 <arg>-XDcompilePolicy=simple</arg>
-                <arg>-Xplugin:ErrorProne -XepExcludedPaths:${project.build.directory}/generated-sources/.* ${errorProneFlags}</arg>
+                <arg>-Xplugin:ErrorProne -XepExcludedPaths:\Q${project.build.directory}\E.* ${errorProneFlags}</arg>
```

Comment:
Curious why this PR and not other encountered trouble?

Reply:
Because @Judge73 is the first developer to run the build with QA on Windows in years (if not ever). He was trying to get QA running on Windows and I've helped him find a way to address the build failure that came with it.

Reply:
Maybe because other people didn't run QA builds on windows without windows subsystem for linux (guess).